crates/rspack_plugin_css/src/dependency/url.rs

```diff
@@ -75,7 +65,7 @@ impl Dependency for CssUrlDependency {
   }
 
   fn span(&self) -> Option<ErrorSpan> {
```

Comment:
Do we still have to keep `ErrorSpan`? Wouldn't `RealDependencyLocation` be enough?

Reply:
This is a huge historical legacy issue, and I plan to address it in the next few PRs.

Reply:
Thanks for the clarification.

Reply:
Would you please rebase and resolve the conflict? I would happy to get this merged.